<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Registro de datos</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID : </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="id_lineEdit"/>
           </item>
           <item row="0" column="4" rowspan="13">
            <widget class="QPlainTextEdit" name="salida"/>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Origen en x : </string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="origen_x_spinBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Origen en y : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="origen_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Destino x : </string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="destino_x_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Destino y : </string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QSpinBox" name="destino_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Velocidad : </string>
             </property>
            </widget>
           </item>
           <item row="5" column="2" colspan="2">
            <widget class="QLineEdit" name="velocidad_lineEdit"/>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Color (RGB) </string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>red</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>green</string>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QSpinBox" name="green_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>blue</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QSpinBox" name="blue_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="4">
            <widget class="QPushButton" name="agregar_final_pushButton">
             <property name="text">
              <string>Agregar al final</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="4">
            <widget class="QPushButton" name="agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar al inicio</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="4">
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <widget class="QPushButton" name="buscar_tabla_pushButton">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>330</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="buscar_tabla_lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>571</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Escriba la ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mostrar_tabla_pushButton">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>330</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tabla">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>751</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="distancia_descendente_pushButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>360</y>
          <width>241</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Distancia (Descendiente)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="id_ascendente_pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>241</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ID (Ascendente)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="velocidad_ascendente_pushButton">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>360</y>
          <width>241</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Velocidad (Ascendente)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Grafica</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar_pushButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Limpiar</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="action_abrir"/>
    <addaction name="action_guardar"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="action_busqueda_profundidad_y_amplitud"/>
    <addaction name="action_algoritmo_prim"/>
    <addaction name="action_algoritmo_kruskal"/>
    <addaction name="action_algoritmo_dijkstra"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_abrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_guardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_busqueda_profundidad_y_amplitud">
   <property name="text">
    <string>Busqueda Profundidad y Aomplitud</string>
   </property>
  </action>
  <action name="action_busqueda_amplitud">
   <property name="text">
    <string>Busqueda Amplitud</string>
   </property>
  </action>
  <action name="action_algoritmo_prim">
   <property name="text">
    <string>Algoritmo Prim</string>
   </property>
  </action>
  <action name="action_algoritmo_kruskal">
   <property name="text">
    <string>Algoritmo Kruskal</string>
   </property>
  </action>
  <action name="action_algoritmo_dijkstra">
   <property name="text">
    <string>Algoritmo Dijkstra</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
